tpch/queries/q14.py

```diff
@@ -1,16 +1,11 @@
 from __future__ import annotations
 
 from datetime import datetime
-from typing import TYPE_CHECKING
 
 import narwhals as nw
 
-if TYPE_CHECKING:
-    from narwhals.typing import FrameT
 
-
-@nw.narwhalify
-def query(line_item_ds: FrameT, part_ds: FrameT) -> FrameT:
+def query(line_item_ds: nw.LazyFrame, part_ds: nw.LazyFrame) -> nw.LazyFrame:
```

Comment:
is the FrameT annotation not correct?

Reply:
We are guaranteed to pass a lazyframe (due to scan_parquet) - happy to rollback though

Reply:
Everything inside each query is supported for both eager and lazy so I think FrameT would be better

Reply:
Nice, diff is much simpler now!